docs/Getting_Started.md

```diff
@@ -0,0 +1,178 @@
+### Add LLVM repo to apt
+
+```bash
+wget https://apt.llvm.org/llvm.sh
+chmod +x llvm.sh
+sudo ./llvm.sh 
+```
+
+### Install Base Packages
+
+```bash
+
+sudo apt install build-essential libzstd1 libzstd-dev libncurses5-dev libzstd-dev zlib1g-dev cmake
+
+sudo apt-get install ocl-icd-libopencl1 
+```
+
+### Install LLVM + Clang + SPIR-V Tools
+
+```bash
+sudo apt install llvm-17 llvm-17-dev libclang-17-dev
```

Comment:
FYI, this does not work unless one runs `sudo ./llvm.sh 17` first. the `llvm.sh` by default installs the latest stable LLVM (18 now).